### PR TITLE
chore(layers-panel): added insert-scene-panel

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
Need to be landed with - https://github.com/visgl/loaders.gl/pull/2273

### CODE REVIEW

File: src/components/comparison/layers-panel/layers-panel.tsx [hunk 1]
```diff
@@ -235,13 +288,37 @@ export const LayersPanel = ({
               />
             </PanelWrapper>
           )}
-
-          {showInsertPanel && (
+          {showExistedSceneWarning && (
+            <PanelWrapper ref={(element) => setWarningNode(element)}>
+              <WarningPanel
+                title={EXISTING_AREA_WARNING}
```
Comment (belom88) [inline src/components/comparison/layers-panel/layers-panel.tsx:294]: The conten of the warning is the same as for `showExistedLayerWarning`. We could use `showExistedLayerWarning` and its warning for the insert scene.

File: src/components/comparison/layers-panel/layers-panel.tsx [hunk 2]
```diff
@@ -161,7 +173,48 @@ export const LayersPanel = ({
...
+
+    const children = webScene.layers.map((child) => ({
+      id: child.id,
+      name: child.title,
+      url: child.url,
+    }));
+
+    if (!children.length) {
+      setShowSceneInsertPanel(false);
+      setShowNoSupportedLayersInSceneWarning(true);
```
Comment (belom88) [inline src/components/comparison/layers-panel/layers-panel.tsx:204]: That was about another logic. If the scene has at least one unsupported layer, the warning should be shown.
If the webscene doesn't contain the data type of children, we need to load layer.json of each child to check.

File: src/components/comparison/layers-panel/layers-panel.tsx [hunk 3]
```diff
@@ -110,6 +112,9 @@ const CloseButtonWrapper = styled.div`
 const EXISTING_AREA_WARNING =
   "You are trying to add an existing area to the map";
 
+const NOT_SUPPORTED_LAYERS_WARNING = "No available supported layers in scene";
+// TODO handle not supported CRS.
+
```
Comment (belom88) [inline src/components/comparison/layers-panel/layers-panel.tsx:117]: Strange comment

**Review by belom88 — CHANGES_REQUESTED** [review]

**Review by belom88 — APPROVED** [review]

**Comment by dryabinin-actionengine** [comment]
Closed because of https://github.com/visgl/loaders.gl-showcases/pull/194

**Pull request closed without merging**